app/controllers/clearance/passwords_controller.rb

```diff
@@ -5,17 +5,20 @@ class Clearance::PasswordsController < Clearance::BaseController
   skip_before_filter :authorize, only: [:create, :edit, :new, :update]
   before_filter :forbid_missing_token, only: [:edit, :update]
   before_filter :forbid_non_existent_user, only: [:edit, :update]
+  before_filter :forbid_expired_password_reset, only: [:edit, :update]
 
   def create
     if user = find_user_for_create
-      user.forgot_password!
-      deliver_email(user)
+      password_reset = PasswordReset.new
+      password_reset.user = user
+      password_reset.save
```

Comment:
It's been a while since I've Rails'ed, but there's a better way to do this, right? It's a one-way association, i.e. there's only a `belongs_to` association in `PasswordReset`, no `has_many` from `User`.